### DISCUSSION TITLE
Naming for the *attribute* to enable "light dismiss for dialog"

### DISCUSSION THREAD

**Discussion by mfreed7** [openui/open-ui#960] [Polls]
See https://github.com/openui/open-ui/discussions/950 for the first incarnation of this poll. After some [additional discussion in WHATWG](https://github.com/whatwg/html/issues/9373#issuecomment-1830376481), we've coalesced around an attribute with multiple values, such as:

- `<dialog attribute_name=none>` - no automatic closing at all
- `<dialog attribute_name=closerequest>` - closes via ESC (or any such close signal)
- `<dialog attribute_name=any>` - closes via ESC *or* clicking outside the bounds of the dialog

This poll is about "`attribute_name`" in the above list. What should that attribute be called?

**Comment by GrimLink** [comment]
In general, I would recommend using `closedby` as it is the most concise, memorable, and easy-to-understand option. However, if you want to emphasize the idea of initiating a closing mechanism, then `closetrigger` is a good choice. Avoid using `autoclose` unless you specifically want to imply that the closing process occurs autonomously.

**Comment by Link2Twenty** [comment]
Would these triggers fire both close and cancel? Using the close keyword makes it feel like cancel wouldn't be fired, maybe dismissby?
